internal/cli/opsmanager/metrics/disks_describe.go

```diff
@@ -39,14 +39,19 @@ func (opts *DisksDescribeOpts) initStore() error {
 	return err
 }
 
+var diskMetricTemplate = `NAME	UNITS	TIMESTAMP		VALUE{{range .ProcessMeasurements.Measurements}}  {{if .DataPoints}}
+{{ $name := .Name }}	{{ $unit := .Units }}{{range .DataPoints}}	
```

Comment:
you have an empty line every now and them due to this line, you can move it to the line above or do something like
```suggestion
{{- $name := .Name }}{{- $unit := .Units }}{{- range .DataPoints}}	
```
the dash(-) removes the code block as a space